fl4health/clients/constrained_fenda_client.py

```diff
@@ -0,0 +1,251 @@
+from logging import WARNING
+from pathlib import Path
+from typing import Dict, Optional, Sequence, Tuple
+
+import torch
+import torch.nn as nn
+from flwr.common.logger import log
+from flwr.common.typing import Config
+
+from fl4health.checkpointing.client_module import ClientCheckpointModule
+from fl4health.clients.basic_client import BasicClient, TorchInputType
+from fl4health.losses.fenda_loss_config import ConstrainedFendaLossConfig
+from fl4health.model_bases.fenda_base import FendaModelWithFeatureState
+from fl4health.parameter_exchange.layer_exchanger import FixedLayerExchanger
+from fl4health.parameter_exchange.parameter_exchanger_base import ParameterExchanger
+from fl4health.utils.losses import EvaluationLosses, LossMeterType
+from fl4health.utils.metrics import Metric
+
+
+class ConstrainedFendaClient(BasicClient):
```

Comment:
why don't we inherit this from fenda client instead of basic client?

Reply:
Yup we can, will move over.